_docs/data-model/v1/upload.md

```diff
@@ -55,6 +59,48 @@ At some point, we may support real-time monitoring of devices, in which case the
 
 The Tidepool platform supports the idea that a user other than the data owner may have upload permissions (for example, a clinic or a parent). The byUser field should be the actual userId of the user doing the upload. This field will be stored in encrypted form in the actual data stream as it constitutes PII.
 
+## device information
+
+### deviceType
+This is the broad category of the device. In general, it should be one of the following:
+
+* `"pump"` (for insulin pumps)
+* `"cgm"` (for continuous glucose monitors)
+* `"pump-cgm"` (for pumps integrated with a cgm)
+* `"bgm"` (for blood glucose meters that are not continuous meters)
```

Comment:
How is something like the Abbott Libre going to fall into this taxonomy? It's not a CGM and not a BGM.

Also, I'd prefer `insulin-pump` to just `pump` and maybe `integrated-cgm` instead of `pump-cgm`.